apps/deploy-web/src/pages/api/auth/[...auth0].ts

```diff
@@ -21,6 +22,15 @@ export default handleAuth({
           try {
             // TODO: Fix for console
             const user_metadata = session.user["https://console.akash.network/user_metadata"];
+            const headers = new AxiosHeaders({
+              Authorization: `Bearer ${session.accessToken}`
+            });
+
+            const anonymousId = req.headers["x-anonymous-user-id"];
+
+            if (anonymousId) {
+              headers.set("X-ANONYMOUS-USER-ID", anonymousId);
+            }
```

Comment:
Should they be both the same format?

Reply:
It's lowercased somewhere along the way anyway. But prob makes sense to make it lowercase everywhere to avoid confusion